### DISCUSSION TITLE
[Python] Disable and enable display on

### DISCUSSION THREAD

**Discussion by r57zone** [OnionUI/Onion#1464] [Q&A]
Hi, is there a way to turn off the display from Python 2.7?
(to save energy for applications such as players)
Thanks

**Comment by XK9274** [comment]
Not sure re: Python, but, you could just push shell commands in:

`echo 0 > /sys/class/pwm/pwmchip0/pwm0/duty_cycle` for off

`echo 100 > /sys/class/pwm/pwmchip0/pwm0/duty_cycle` for on

It'll turn the display off until you press a button in most cases.

You can test these on SSH/Telnet.

Or, you can use GPIO:

Turn off with:
```
/ # echo 4 > /sys/class/gpio/export
/ # echo in > /sys/devices/gpiochip0/gpio/gpio4/direction
```

Turn on with:

```
/ # echo out > /sys/devices/gpiochip0/gpio/gpio4/direction
/ # echo 1 > /sys/devices/gpiochip0/gpio/gpio4/value
```

**Reply by r57zone** [reply]
Thank you
